[RLlib; docs] Docs do-over (new API stack): Update "new API stack" header include. (#49465)
ray-project/ray: doc/source/_includes/rllib/new_api_stack.rst

.. note::

    Ray 2.40 uses :doc:`RLlib's new API stack </rllib/rllib-new-api-stack>` by default.

>     The team has mostly completed transitioning algorithms, example scripts, and

Reviewer: ```suggestion
    The Ray team has mostly completed transitioning algorithms, example scripts, and
```

    documentation to the new code base.

>     In case you are still using the old API stack, :doc:`see here </rllib/new-api-stack-migration-guide>` for more details on how to migrate.

Reviewer 2: I still think we should give a certain date - maybe even just a month in 2025 in which we do not support anymore the old stack.

Author: We need to see. Can't do that so easily b/c of a certain customer.

>     In case you are still using the old API stack, :doc:`see here </rllib/new-api-stack-migration-guide>` for more details on how to migrate.

Reviewer: ```suggestion
    If you're still using the old API stack, see :doc:`New API stack migration guide </rllib/new-api-stack-migration-guide>` for details on how to migrate.
```
